.internal-ci/helm/fog-services/values.yaml

```diff
@@ -41,18 +41,24 @@ ingress:
   enabled: true
 
 #### Fog View
-fogViewStores:
+fogViewShardRangeGenerator:
+  # The dev options are set in the github workflow deployment.
+  # This is just for reference.
+  # dev:
+  #   shardSize: 25_000
```

Comment:
Looking at .github/workflows/mobilecoin-workflow-dev-deploy.yaml
the values seem to be 10_000 for size and 5_000 for exceed by

Reply:
I'll fix that reference